hexrd/ui/hand_drawn_mask_dialog.py

```diff
@@ -134,6 +134,9 @@ def line_finished(self):
         self.add_line()
 
     def button_pressed(self, event):
+        if event.dblclick:
+            self.accepted()
+            self.ui.close()
```

Comment:
Right now, this `event.dblclick` includes double left-click, double middle-click, and double right-click. Do we want this to be just for double left-click?

Also, we should return after `self.ui.close()`, otherwise, the code below this continues to run (and double right-click produces some errors).

Reply:
Right now I've updated this to only handle left-click double-click but that can be changed if it should be. Return statement was added as well.